api/android/api/src/com/samsung/android/nnstreamer/TensorsInfo.java

```diff
@@ -242,5 +256,39 @@ public void setDimension(@NonNull int[] dimension) {
         public int[] getDimension() {
             return this.dimension;
         }
+
+        public int getSize() {
+            int size = 0;
+
+            switch (this.type) {
+                case NNStreamer.TENSOR_TYPE_INT32:
+                case NNStreamer.TENSOR_TYPE_UINT32:
+                case NNStreamer.TENSOR_TYPE_FLOAT32:
+                    size = 4;
+                    break;
+                case NNStreamer.TENSOR_TYPE_INT16:
+                case NNStreamer.TENSOR_TYPE_UINT16:
+                    size = 2;
+                    break;
+                case NNStreamer.TENSOR_TYPE_INT8:
+                case NNStreamer.TENSOR_TYPE_UINT8:
+                    size = 1;
+                    break;
+                case NNStreamer.TENSOR_TYPE_FLOAT64:
+                case NNStreamer.TENSOR_TYPE_INT64:
+                case NNStreamer.TENSOR_TYPE_UINT64:
+                    size = 8;
+                    break;
+                default:
+                    /* unknown type */
```

Comment:
```throw```?

Reply:
In getTensorSize(), added exception case when returned size is 0 or minus value.

Reply:
OK